README.md

```diff
@@ -194,6 +194,90 @@ export function a11yAuditIf(contextSelector, axeOptions) {
 }
 ```
 
+#### Setting Options
+
+You can provide options to axe-core for your tests using the `setRunOptions` API. Options can
+be set a few ways:
+
+Globally:
+
+```javascript
+// tests/test-helper.js
+import { setRunOptions } from 'ember-a11y-testing';
+
+setRunOptions({
+  rules: {
+    region: { enabled: true },
+  },
+  checks: {
+    'color-contrast': {
+      options: {
+        noScroll: true,
+      },
+    },
+  },
+});
+```
+
+Test module level:
+
+```javascript
+import { module, test } from 'qunit';
+import { setRunOptions } from 'ember-a11y-testing';
+
+module('some test module', function (hooks) {
+  hooks.before(function () {
```

Comment:
Needs to be `beforeEach` (this is because we reset after each test):

```suggestion
  hooks.beforeEach(function () {
```

Reply:
Ya I originally was going to do this, but thought it made more sense to do it once rather than for each test invoked. I supposed it doesn't matter, but it's not clear to me what `getContext().owner` would return in `hooks.before`. What would it be?